nat-lab/bin/netcat.py

```diff
@@ -0,0 +1,192 @@
+import argparse
+import errno
+import select
+import socket
+import sys
+import time
+from typing import TextIO, List, Union
+
+UDP_SCAN_COUNT: int = 5
+RECV_SIZE = 4096
+
+
+class NetCat:
+    """A netcat clone written in Python"""
```

Comment:
Alternative would have been to build BSD netcat from source on macOS, but then we would still need to find a solution for Windows